sakip_sabanci_config.rb

```diff
@@ -3,13 +3,15 @@
 require 'traject_plus'
 require 'dlme_json_resource_writer'
 require 'macros/dlme'
-require 'macros/bnf'
+require 'macros/oai'
+require 'macros/ssm'
 
 extend Macros::DLME
+extend Macros::OAI
 extend TrajectPlus::Macros
 extend TrajectPlus::Macros::Xml
 
-NS = { xmlns: 'http://www.openarchives.org/OAI/2.0/' }.merge(Macros::BNF::NS).freeze
+NS = { xmlns: 'http://www.openarchives.org/OAI/2.0/' }.merge(Macros::OAI::NS).freeze
```

Comment:
There is no need to have this line if you convert all the extract_xml calls into extract_oai calls.